iceoryx_posh/include/iceoryx_posh/internal/popo/publisher.inl

```diff
@@ -51,58 +51,59 @@ inline cxx::expected<AllocationError> Publisher<T, base_publisher_t>::publishRes
 
     return loanSample().and_then([&](auto& sample) {
         c(sample.get(), std::forward<ArgTypes>(args)...);
-        sample.publish();
+        publish(std::move(sample));
```

Comment:
Why are you using this approach instead of `sample.publish`. Is `sample.publish` going to be removed?

Reply:
I guess that's an oddment from an intermediate state when `sample.publish` didn't work. Will fix it

Reply:
same in line 63